settings.d/dynflow.yml.example

```diff
@@ -0,0 +1,2 @@
+---
+:enabled: false
```

Comment:
If there are no other manual options we can make it enabled by default, btw user will have to restart the proxy manually anyway. OTOH if other plugins are disabled by default, I'd keep it same for consistency. IIRC the installer usually configures these files so this case applies only when it's installed manually from package.